anyway/utilities.py

```diff
@@ -36,16 +36,16 @@ def init_flask():
     )
     app.config.from_object(config)
     app.config["BABEL_TRANSLATION_DIRECTORIES"] = os.path.join(_PROJECT_ROOT, "translations")
-    if os.environ.get("PROXYFIX_X_FOR"):
+    if secrets.get("PROXYFIX_X_FOR"):
```

Comment:
Is it really a secret?

Reply:
I don't know, not clear what's considered a secret and what isn't. 
Asking @atalyaalon